lib/widgets/compose_box.dart

```diff
@@ -19,8 +19,8 @@ import 'icons.dart';
 import 'store.dart';
 import 'theme.dart';
 
-const double _inputVerticalPadding = 8;
```

Comment:
Hmm, that would affect other `IconButton`s including the back button on the AppBar. Perhaps we should do that in a cleanup PR. Refactored the theme setup so that the splash effect is properly disabled.